packages/ui/Scroller/Scroller.js

```diff
@@ -49,6 +49,8 @@ const Scroller = (props) => {
 		horizontalScrollbarProps
 	} = useScroll(props);
 
+	delete scrollContentProps.scrollMode;
```

Comment:
It would be better to remove the `scrollMode` in the `ScrollerBasic`.

Reply:
VirtualList bleeds the `scrollMode` prop. The VirtualList also need to remove it.